CHANGELOG.md

```diff
@@ -70,13 +71,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Features
 
+* [#2086] Pool-incentives: remove tx command, add proposal-specific flags to proposal-submit commands 
```

Comment:
Let's try to avoid duplication
```suggestion
```

Reply:
@catShaark please ping me once this is addressed - I think this is ready for merge

Reply:
@catShaark, I've committed your suggestion